package.json

```diff
@@ -17,32 +17,32 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^14.2.12",
-    "@angular/common": "^14.2.12",
-    "@angular/compiler": "^14.2.12",
-    "@angular/core": "^14.2.12",
-    "@angular/elements": "^14.2.12",
-    "@angular/forms": "^14.2.12",
-    "@angular/platform-browser": "^14.2.12",
-    "@angular/platform-browser-dynamic": "^14.2.12",
-    "@angular/router": "^14.2.12",
+    "@angular/animations": "^15.2.9",
+    "@angular/common": "^15.2.9",
+    "@angular/compiler": "^15.2.9",
+    "@angular/core": "^15.2.9",
+    "@angular/elements": "^15.2.9",
+    "@angular/forms": "^15.2.9",
+    "@angular/platform-browser": "^15.2.9",
+    "@angular/platform-browser-dynamic": "^15.2.9",
+    "@angular/router": "^15.2.9",
     "@laserfiche/lf-js-utils": "^4.0.3",
     "@laserfiche/lf-repository-api-client": "^1.1.3",
-    "@laserfiche/lf-ui-components": "^14.1.4",
-    "@laserfiche/lf-ui-components-services": "^14.1.0",
-    "rxjs": "^6.6.0",
+    "@laserfiche/lf-ui-components": "^15.0.1",
+    "@laserfiche/lf-ui-components-services": "^15.0.0",
+    "rxjs": "7.8",
```

Comment:
was this an update by angular?

Reply:
No, but lf-ui-components uses rxjs: "^7.8.0"
And I was getting some dependency errors here before updating this. 
Perhaps I should correct it to "^7.8.0" instead of just "7.8" since I installed the particular version to test